homu/main.py

```diff
@@ -699,6 +713,9 @@ def main():
         repo_cfgs[repo_label] = repo_cfg
         repo_labels[repo_cfg['owner'], repo_cfg['name']] = repo_label
 
+        if not 'try' in repo_cfgs[repo_label]:
+            repo_cfgs[repo_label]['try'] = []
```

Comment:
IMO automated changes to the configuration should be done with caution; that is, it might be better to use `repo_cfg.get('try', [])` in the retrieving site, rather than changing the original value. Also, it is possible that the configuration is injected on-the-fly from Homu Cloud (homu.io), which may not necessarily provide the `try` reviewer list.